From a602f7364666fd44e0846c606ff0fe64bc3f84c7 Mon Sep 17 00:00:00 2001
From: Matt Westcott <matt@west.co.tt>
Date: Fri, 26 Jun 2020 15:13:15 +0100
Subject: [PATCH 07/11] Fix pytest installation instructions

Since the removal of requirements_test.txt (in 2a0e8fb756d7c9dfce5f66e2b3cf68274dbc96d5 and 63610e0fd83d2fbb4126963bbe22b5b0a76f6104) it is no longer the case that `pip install -r requirements.txt` will install pytest.
---
 docs/source/tests.rst | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/source/tests.rst b/docs/source/tests.rst
index 2218bed..f75d11e 100644
--- a/docs/source/tests.rst
+++ b/docs/source/tests.rst
@@ -7,12 +7,11 @@ recommended that you run and update the test suite when you send patches.
 py.test
 -------
 
-You will need `pytest`_ to run the test suite. It's included with the
-development dependencies:
+You will need `pytest`_ to run the test suite:
 
 .. code-block:: console
 
-    $ pip install -r requirements.txt
+    $ pip install pytest
 
 Then just run the test suite:
 
-- 
2.25.4

